src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExampleDriver.java

```diff
@@ -57,69 +65,30 @@ public static void main(String [] args) throws Exception {
-                                                    "Punk"),
-                                           new Song(6L,
-                                                    "The Audacity Of Hype",
-                                                    "Jello Biafra And The Guantanamo School Of "
-                                                    + "Medicine",
-                                                    "Three Strikes",
-                                                    "Punk"),
-                                           new Song(7L,
-                                                    "Licensed to Ill",
-                                                    "The Beastie Boys",
-                                                    "Fight For Your Right",
-                                                    "Hip Hop"),
-                                           new Song(8L,
-                                                    "De La Soul Is Dead",
-                                                    "De La Soul",
-                                                    "Oodles Of O's",
-                                                    "Hip Hop"),
-                                           new Song(9L,
-                                                    "Straight Outta Compton",
-                                                    "N.W.A",
-                                                    "Gangsta Gangsta",
-                                                    "Hip Hop"),
-                                           new Song(10L,
-                                                    "Fear Of A Black Planet",
-                                                    "Public Enemy",
-                                                    "911 Is A Joke",
-                                                    "Hip Hop"),
-                                           new Song(11L,
-                                                    "Curtain Call - The Hits",
-                                                    "Eminem",
-                                                    "Fack",
-                                                    "Hip Hop"),
-                                           new Song(12L,
-                                                    "The Calling",
-                                                    "Hilltop Hoods",
-                                                    "The Calling",
-                                                    "Hip Hop")
-
-    );
+
+    // Read comma-delimited file of songs into Array
+    final List<Song> songs = new ArrayList<>();
+    final String SONGFILENAME= "song_source.csv";
+    final InputStream inputStream = KafkaMusicExample.class.getClassLoader().getResourceAsStream(SONGFILENAME);
+    final InputStreamReader streamReader = new InputStreamReader(inputStream, UTF_8);
+    final BufferedReader br = new BufferedReader(streamReader);
+    try {
```

Comment:
Nit: update to "try-with-resource" clause:
```
try (final BufferedReader br = new BufferedReader(streamReader)) {
  ...
}
```
This allows to omit `catch` and `final` block .